packages/react-dom/npm/static.browser.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
Do these not use the DCE check code because they are not client and won’t always be bundled?

Reply:
It matters less on the server so not worth the code. If you're setting up a server environment you are down a deeper rabbit hole anyway.